labcontrol/gui/templates/library_pooling_16S.html

```diff
@@ -71,25 +68,17 @@
   /**
    * Determine an appropriate clipping value for the heatmap's maximum value.
    *
-   * @param {Float} plateType String describing the plate type.
    * @returns A maximum expected value for the colorscale of a heatmap.
    *
    */
-  function clippingForLibraryPlateType(plateType) {
-    if (plateType == '16S library prep') {
-      return 15;
-    }
-    else if (plateType == 'shotgun library prep') {
-      return 1000;
-    }
-
-    // should never get here but if it does, then the range is permissive
-    return 10000;
+  function clippingForLibraryPlateType() {
+    // default value ofr 16S
```

Comment:
Should be `for` instead of `ofr`, as far as I can tell

Reply:
Good call.